<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerSearchGui</class>
 <widget class="QDialog" name="SellerSearchGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dokan Online Market</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/img/assets/img/icons/icons8-small-business-96.png</normaloff>:/img/assets/img/icons/icons8-small-business-96.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: #1978AB;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="lb_logo_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>18</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/img/assets/img/logo-white.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>97</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color : #ffffff; 
border-radius: 12px;
padding: 0 0 0 20px;

} </string>
    </property>
    <property name="placeholderText">
     <string>Search </string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>169</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar:vertical {
    border: 2px solid grey;
    background: #32CC99;
    height: 15px;
    margin: 0px 20px 0 20px;
}
QScrollBar::handle:vertical {
    background: white;
    min-width: 20px;
}
QScrollBar::add-line:vertical {
    border: 2px solid grey;
    background: #32CC99;
    width: 20px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>329</width>
       <height>399</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="parentFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>131</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color : #ffffff; 
border-radius: 12px;
background-color: #E3BA5A;
	color: rgb(255, 255, 255);
} </string>
    </property>
    <property name="text">
     <string>click here</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_NoProduct">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
letter-spacing: 1px;
color: #a6a6a6;
}</string>
    </property>
    <property name="text">
     <string>No Prodects Found</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
